server/api/observation/observation-create.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response } from 'express';
+import { Observation } from './observation.model';
+import { ObservationType } from './type/observation-type.model';
+
+class ObservationCreateController {
+  async createObservation(req: Request, res: Response) {
```

Comment:
We tend to use more Typescript-esque conventions. These are the non-typed versions of Request/Response - we should probably follow similar conventions to the controllers and their action methods.

Reply:
Will be fixed as part of: https://app.asana.com/0/1198942245525292/1200516880894435/f